pkgs/development/python-modules/execnet/default.nix

```diff
@@ -57,11 +57,11 @@ buildPythonPackage rec {
 
   __darwinAllowLocalNetworking = true;
 
-  meta = with lib; {
-    changelog = "https://github.com/pytest-dev/execnet/blob/v${version}/CHANGELOG.rst";
+  meta = {
     description = "Distributed Python deployment and communication";
     homepage = "https://execnet.readthedocs.io/";
-    license = licenses.mit;
-    maintainers = with maintainers; [ ];
+    changelog = "https://github.com/pytest-dev/execnet/blob/v${version}/CHANGELOG.rst";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ getchoo ];
```

Comment:
I would rather keep meta attributes sorted alphabetically.

Reply:
i actually based this off of the order in which they're listed in https://ryantm.github.io/nixpkgs/stdenv/meta/#chap-meta. this obviously isn't an official standard by any means, but imo makes a lot of sense in the context of what the reader actually wants to see and matches many other packages i've seen

regardless, i don't think this should really be a blocker